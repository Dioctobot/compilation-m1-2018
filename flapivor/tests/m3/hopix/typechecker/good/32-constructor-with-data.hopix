type Option = Some (Int) | None

val none : Option = None<>

val some42 : Option = Some<> (42)