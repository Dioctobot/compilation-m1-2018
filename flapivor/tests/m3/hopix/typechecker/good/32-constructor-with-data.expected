none : Option
some42 : Option
